cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/ExtractTemplatesUtil.java

```diff
@@ -128,6 +141,31 @@ private static void processJar(Path destinationPath) throws IOException {
 
   }
 
+  /**
+   * Unpacks the source CobiGen_Templates Jar and creates a new CobiGen_Templates folder structure at
+   * $destinationPath/CobiGen_Templates location
```

Comment:
The location does not match with the new structure of the templates directory

Reply:
Adjusted.